### PR TITLE
Video Quality (full) Screen

### PR DESCRIPTION
**Pull request by mistermendez**

### CODE REVIEW

File: js/components/closeButton.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ var CloseButton = React.createClass({
     event.stopPropagation();
     event.cancelBubble = true;
     event.preventDefault();
-    return false;
   },
```
Comment (mistermendez) [inline js/components/closeButton.js]: Resolve warning - returning `false` from an event handler is deprecated

File: js/controller.js [hunk 2]
```diff
@@ -823,6 +844,17 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
...
 
+    sendVideoQualityChangeEvent: function(selectedContentData) {
+      this.state.videoQualityOptions.selectedBitrate = {
+        "id": selectedContentData.id
+      };
+      this.state.screenToShow = CONSTANTS.SCREEN.LOADING_SCREEN;
+      this.renderSkin();
+      this.mb.publish(OO.EVENTS.PAUSE);
+      this.mb.publish(OO.EVENTS.SET_EMBED_CODE, this.state.assetId);
+      this.mb.publish(OO.EVENTS.BITRATE_CHANGED, selectedContentData.selectedBitrate);
```
Comment (ksunika) [inline js/controller.js]: out of the events we have in constants, SET_TARGET_BITRATE seems to be more appropriate

File: js/controller.js [hunk 3]
```diff
@@ -823,6 +844,13 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.DISCOVERY_API.SEND_DISPLAY_EVENT, eventData);
     },
 
+    sendVideoQualityChangeEvent: function(selectedContentData) {
+      this.state.videoQualityOptions.selectedBitrate = {
+        "id": selectedContentData.id
+      };
+      this.mb.publish(OO.EVENTS.SET_TARGET_BITRATE, selectedContentData.selectedBitrate);
```
Comment (ksunika) [inline js/controller.js]: I know it's only a mock, but selectedContentData only has 'id' property here, maybe we should publish selectedContentData.id then.

**Comment by mistermendez** [comment]
@ksunika updated to publish id

File: js/controller.js [hunk 4]
```diff
@@ -823,6 +844,13 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.DISCOVERY_API.SEND_DISPLAY_EVENT, eventData);
     },
 
+    sendVideoQualityChangeEvent: function(selectedContentData) {
+      this.state.videoQualityOptions.selectedBitrate = {
+        "id": selectedContentData.id
+      };
+      this.mb.publish(OO.EVENTS.SET_TARGET_BITRATE, selectedContentData.selectedBitrate.id);
```
Comment (ksunika) [inline js/controller.js]: @mistermendez, doesn't work this way. Need to change selectedContentData.selectedBitrate.id to selectedContentData.id.

**Comment by mistermendez** [comment]
:stuck_out_tongue_winking_eye:

**Comment by ksunika** [comment]
looks good!

**Pull request merged**